index.js

```diff
@@ -1,5 +1,69 @@
 'use strict';
 
+var superagent = require('superagent');
+
+global.D2LAccessTokenExpiresAt = 0;
+
+function now() {
+	return Date.now()/1000 | 0;
```

Comment:
Cool makes sense. Yeah them messing with their clock would screw up a few things on the web likely.